website/src/react-native/showcase.js

```diff
@@ -445,6 +445,15 @@ var pinned = [
     infoTitle: 'React Native at Walmart Labs',
     defaultLink: 'https://itunes.apple.com/us/app/walmart-app-shopping-savings/id338137227?mt=8',
   },
+  {
```

Comment:
Please take a look at the comment in line 334. This is not the right place to add this app.